doc/manual/tuto.md

```diff
@@ -0,0 +1,187 @@
+---
+slug: tutorial
+---
+
+# Managing users
+
+In this tutorial, you will learn how to use Nickel to manage a list of
+users, and then export it as a YAML file for a program consuming it.
+
+## Step 1: Install nickel
+
+The first step is to obtain nickel on your system. There are different
+methods to install Nickel, they are all covered on the [Getting
+started](https://nickel-lang.org/getting-started/#getting-started)
+page of Nickel project website.
+
+## Step 2: Think about the schema
+
+The YAML we want to produce has different fields with different types
+(lists, booleans, strings). Here is a sample of what we need:
+
+```yaml
+users:
+  - name: Aisha
+    is-admin: true
+    ssh-keys:
+      - AAAAApqCA8oKAB5S/47f...... aisha@work
+  - name: Violet
+    extra-groups:
+      - accounting
+```
+
+We can spot attributes such as `name`, `ssh-keys`, `is-admin` or
```

Comment:
Just to be consistent with the documentation and the naming in Nickel in general (attributes is Nix terminology). I think we even use `attribute` sometimes for metadata like `optional` or `default`.

```suggestion
We can spot fields such as `name`, `ssh-keys`, `is-admin` or
```